src/Development/IDE/GHC/Util.hs

```diff
@@ -2,61 +2,71 @@
 -- SPDX-License-Identifier: Apache-2.0
 
 -- | General utility functions, mostly focused around GHC operations.
-module Development.IDE.GHC.Util(
+module Development.IDE.GHC.Util
```

Comment:
Why did you reformat this file now? That makes diff noisy and `git blame` harder to use.

Reply:
I guess my editor did - will undo

Reply:
Thanks! CI is failing now due to #479 being merged (CI tests the merge commit, so you need to rebase to reproduce).